holoviews/operation/element.py

```diff
@@ -1270,7 +1307,7 @@ def _process(self, element, key=None):
         sort_dims, dendros = [], {}
         for d in self.p.adjoint_dims:
             ddata = self._compute_linkage(dataset, d, self.p.main_dim)
-            order = [ddata["ivl"].index(v) for v in dataset.dimension_values(d)][::-1]
+            order = [ddata["ivl"].index(v) for v in dataset.dimension_values(d)]
```

Comment:
Removing this makes the output aligned with how scanpy looks (other than a reversed ordering of the axis):

![image](https://github.com/user-attachments/assets/d8ec9abb-090a-4450-aa70-81add80fda2c)

![image](https://github.com/user-attachments/assets/c9a3c5d9-b52e-46da-8966-d98e6d171970)